src/main/java/org/mule/extension/validation/internal/ValidationStrategies.java

```diff
@@ -36,4 +36,18 @@ public void all(@AllowedStereotypes(ValidatorStereotype.class) Chain validations
       throws MultipleValidationException {
     // implemented as privileged operation in AllOperationExecutor
   }
+
+  /**
+   * Perform a list of nested validation operations and informs only one {@code VALIDATION:MULTIPLE} error which
+   * summarizes all of the found errors (if all failed).
+   *
+   * @param validations the nested validation operations
+   * @throws MultipleValidationException if all validators fail and {@code throwsException} is {@code true}
+   */
+  @Throws(AllErrorType.class)
+  public void any(@AllowedStereotypes(ValidatorStereotype.class) Chain validations,
```

Comment:
not void

Reply:
https://github.com/mulesoft/mule-extensions-api/blob/master/mule-extensions-api/src/main/java/org/mule/runtime/extension/api/annotation/param/stereotype/Validator.java#L23